github/feeds.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+// FeedsService provides access to the feeds related functions
+// in the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/feeds/
+type FeedsService struct {
```

Comment:
Done.  It's interesting that their docs place this in /activity/feeds/ but the actual RESTful API uses just /feeds.